build-system/dep-check-config.js

```diff
@@ -351,6 +356,62 @@ exports.rules = [
   {
     filesMatching: 'src/**/*.js',
     mustNotDependOn: 'extensions/**/*.js',
+    whitelist: [
```

Comment:
Shouldn't the whitelist be added to this section?

https://github.com/ampproject/amphtml/blob/87a2bdf6c97074585788d3592d8be6c81f61e580/build-system/dep-check-config.js#L207-L211

Reply:
Lolololol.